code/lib/cli/src/initiate.ts

```diff
@@ -275,11 +275,11 @@ async function doInitiate(options: CommandOptions, pkg: Package): Promise<void>
   logger.log(chalk.inverse(`\n ${welcomeMessage} \n`));
 
   // Update notify code.
-  const { default: updateNotifier } = await import('update-notifier');
-  updateNotifier({
-    pkg,
+  const { default: updateNotifier } = await import('simple-update-notifier');
+  await updateNotifier({
+    pkg: pkg as any,
```

Comment:
requires `name` and `string`, which are optional in `PackageJson`.

https://github.com/alexbrazier/simple-update-notifier/blob/394516c457e22cf7c1ae5bd17ea30604f36e231d/src/types.ts#L2
https://github.com/sindresorhus/type-fest/blob/69503987e07762973b9712b6132d0bc268e7ae02/source/package-json.d.ts#L358-L367